app/collections/show.jsx

```diff
@@ -52,19 +52,20 @@ class CollectionPage extends React.Component {
     }
   }
 
-  canCollaborate = () => {
+  canCollaborate() {
     let canCollaborate;
+    let isOwner;
     if (!this.props.user) {
       canCollaborate = false;
+      isOwner = false;
     } else {
       canCollaborate = this.props.roles.some((role) => {
         const idMatch = (role.links.owner.id === this.props.user.id);
         const isCollaborator = role.roles.includes('collaborator');
-
         return isCollaborator && idMatch;
       });
+      isOwner = this.props.user.id === this.props.owner.id;
     }
-    const isOwner = this.props.user.id === this.props.owner.id;
 
     this.setState({ canCollaborate: canCollaborate || isOwner });
   };
```

Comment:
Should there be a semi-colon here?

Reply:
Once #4678 is merged, the linter should work on this file.